ddtrace/opentelemetry/_context.py

```diff
@@ -58,7 +62,9 @@ def get_current(self):
             span = Span(ddactive)
             context = set_span_in_context(span, context)
         elif isinstance(ddactive, DDContext):
-            span_context = OtelSpanContext(ddactive.trace_id or 0, ddactive.span_id or 0, True)
+            tf = TraceFlags.SAMPLED if ddactive._traceflags == "01" else TraceFlags.DEFAULT
+            ts = TraceState.from_header([ddactive._tracestate])
+            span_context = OtelSpanContext(ddactive.trace_id or 0, ddactive.span_id or 0, True, tf, ts)
```

Comment:
Same as above. Ensure traceflags and tracestate from the active context are propagated.